<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayMusicWindow</class>
 <widget class="QMainWindow" name="PlayMusicWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GSP - Musik abspielen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QSlider:groove:horizontal {
border-image: url(:/theme/play/RES/theme/Musik_abspielen/Zeitleiste_Leer.png);
}

QSlider::handle:horizontal {
image: url(:/theme/play/RES/theme/Musik_abspielen/Zeitleiste_Kreis.png);
border-width: 0px;
}

QSlider::sub-page:horizontal {
background-image: url(:/theme/play/RES/theme/Musik_abspielen/Zeitleiste_voll.png);
}

#lblCurrentSong {
font-weight: bold;
color: rgb(71, 71, 71);
font-size: 12px;
}

#cmbPlaylists {
border-image: url(:/theme/play/RES/theme/Musik_abspielen/Dropdown_normal.png);
padding-right: 25px;
padding-left: 10px;
}

#cmbPlaylists:hover {
border-image: url(:/theme/play/RES/theme/Musik_abspielen/Dropdown_hover.png);
}

#cmbPlaylists:on {
border-image: url(:/theme/play/RES/theme/Musik_abspielen/Dropdown_pressed.png);
}

#cmbPlaylists::drop-down {
border-width: 0px;
}

#lblPlayedPlaylist {
font: 12px;
font-weight: bold;
color: rgb(255, 255, 255);
}

#centralwidget {
background-image: url(:/theme/play/RES/theme/Musik_abspielen/BG.png);
}

#lblPlayMusic {
font: 16px;
font-weight: bold;
font-family: arial, verdana, sans-serif;
color: rgb(255, 255, 255);
}

QLabel {
font: 14px &quot;Arial&quot;;
color: rgb(163, 163, 163);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblPlayMusic">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>123</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Musik abspielen</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPlayedPlaylist">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>55</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Abgespielte Playlist:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbPlaylists">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>45</y>
      <width>163</width>
      <height>35</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentSong">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>201</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Kein Song abgespielt</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCoverBeforeBackground">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>115</y>
      <width>154</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Design.qrc">:/theme/play/RES/theme/Musik_abspielen/Cover_klein_Rahmen+Schatten.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentCoverBackground">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>194</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Design.qrc">:/theme/play/RES/theme/Musik_abspielen/Cover_gross_Rahmen+Schatten.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblCoverAfterBackground">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>115</y>
      <width>154</width>
      <height>166</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Design.qrc">:/theme/play/RES/theme/Musik_abspielen/Cover_klein_Rahmen+Schatten.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="lblCurrentCover">
    <property name="geometry">
     <rect>
      <x>262</x>
      <y>112</y>
      <width>190</width>
      <height>190</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblBeforeCover">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>117</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblAfterCover">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>117</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSlider" name="sldPosition">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>670</width>
      <height>15</height>
     </rect>
    </property>
    <property name="maximum">
     <number>0</number>
    </property>
    <property name="pageStep">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="Design.qrc"/>
 </resources>
 <connections/>
</ui>
